tests/DataFixtures/Factory/CountryFactoryTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Tests\DataFixtures\Factory;
+
+use Sylius\Bundle\CoreBundle\DataFixtures\Factory\CountryFactory;
+use Sylius\Component\Addressing\Model\CountryInterface;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\Intl\Countries;
+use Zenstruck\Foundry\Test\Factories;
+use Zenstruck\Foundry\Test\ResetDatabase;
+
+final class CountryFactoryTest extends KernelTestCase
+{
+    use ResetDatabase;
+    use Factories;
+
+    /** @test */
+    function it_creates_country_with_random_code(): void
+    {
+        $country = CountryFactory::new()->create();
+
+        $this->assertInstanceOf(CountryInterface::class, $country->object());
+        $this->assertNotNull($country->getCode());
+        $this->assertTrue(Countries::exists($country->getCode()));
+    }
+
+    /** @test */
+    function it_creates_country_with_custom_code(): void
```

Comment:
```suggestion
    function it_creates_country_with_given_code(): void
```
It's not really a custom code.